src/SerilogTracing/LoggerConfigurationTracingExtensions.cs

```diff
@@ -18,8 +20,10 @@ public static class LoggerConfigurationTracingExtensions
     /// <returns>The logger.</returns>
     public static Logger CreateTracingLogger(this LoggerConfiguration loggerConfiguration, Action<SerilogActivityListenerOptions>? configure = null)
     {
-        var listener = new ActivityListener();
-        var disposeProxy = new DisposeProxy(listener);
+        var activityListener = new ActivityListener();
+        var diagnosticListenerSubscription = DiagnosticListener.AllListeners.Subscribe(new DiagnosticListenerObserver());
+        var disposeProxy = new DisposeProxy(activityListener, diagnosticListenerSubscription);
+        
         var logger = loggerConfiguration
             .WriteTo.Sink(disposeProxy)
```

Comment:
Does Serilog call `Dispose` for you on shutdown?

Reply:
Yes, when the `Logger` is disposed 👍